libraries/postgres/src/util.js

```diff
@@ -0,0 +1,10 @@
+exports.dollarQuote = str => {
+  let i = '';
+  while (true) {
+    const quote = `$${i}$`;
+    if (str.indexOf(quote) === -1) {
+      return quote + str + quote;
+    }
+    i = i + '-';
+  }
+};
```

Comment:
This is like a tongue twister.

Reply:
It's definitely a weird quoting mechanism..